tests/ops/qubit/test_all_qubit_ops.py

```diff
@@ -21,6 +21,23 @@
 from gate_data import I
 
 
+@pytest.mark.parametrize("op_str", qml.ops.qubit.ops)
```

Comment:
We moved away from the design of testing something on all ops - if a new op is added that just doesn't work for the test logic, it is a major nightmare.

I personally prefer tests that do not rely on ops being there at all. I think it is better to test exactly what we use: make a dummy op that overwrites ``compute_matrix`` and check for the dummy op.